README.md

```diff
@@ -42,20 +42,51 @@ Download and install the following tools:
 
 * [Python 3](https://www.python.org/downloads/)
 * [PyCharm](https://www.jetbrains.com/pycharm/)
-* [MiniConda](https://docs.conda.io/en/latest/miniconda.html)
 
 ### Windows users
 
-* Open `miniconda3`.
-* Go to the project path
-* Run `conda create --prefix=.venv python=3.10.0`
-* In order to activate an environment run the following command: `conda activate .\venv\Scripts\activate.bat`
+* Clone git Repository of this project `EA_RESTAURANT`.
+* * Open terminal in pycharm.
+* Run command `git checkout -b dev` and then `git pull origin dev`
+* In Order to create a virtual environment run `python -m venv .venv` 
+* In order to activate an environment run the following command: `.venv\Scripts\activate.bat`
+* In order to set python Interpreter config the one installed inside .venv in pycharm ide.
+* In order to install library dependency run the following command : `pip install -r requirements.txt`
 * In order to update library dependency run the following command : `pip freeze > requirements.txt`
 * In order to set Environment variables file :
   * - Use `.env.local.example` file to create the different environment files: local, dev, test. 
   * - choose under which environment you are going to work using command: `SET ENV=local` for local, `SET ENV=test` for
       testing.
   * - Run the script that you want to run under chosen environment.
+
+### setup Data base:
+* create `POSTGRESQL` database with the name: `ea_restaurant`
+* Run migrations: 
+
+Executing `alembic upgrade head` in a clean database will execute all migrations
+in the orders that migrations were created. Like example image.
+
+![img.png](img.png)
+
+To know more about migrations refer to migrations section.
+
+* Create `MONGODB` database with name: `ea_restaurant` and create a collection with name:`order_status_histories`
```

Comment:
You need to mention what tools you suggest to create a mongo db and also show the steps for the tool you use.

Reply:
Done.